README.md

```diff
@@ -2,9 +2,15 @@
 
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Tietokilta_tietokilta.fi&metric=alert_status)](https://sonarcloud.io/dashboard?id=Tietokilta_tietokilta.fi)
 
+## Requirements
+- Docker
+- NPM
+
 ## Setup
 
 ```
+docker-compose up -d
```

Comment:
I think we might want to comment these lines. This line starts (only) the database and the following ones start the CMS app.